### ISSUE TITLE
Git: Line History causes empty list in v0.6.X

### ISSUE THREAD

**Issue by ole1986** [DonJayamanne/gitHistoryVSCode#486]
Mine does also not work suddenly. When I ask for a line history I get this in the console. Git 2.25.1 on Windows, VScode 1.42.1. No preferences set for the extension.

Seems like there is a commitID or HEAD missing from the command called

```sh
git shortlog -e -s -n HEAD  (completed in 0.245s)
git log -L16,16:projects/gudena-app-ui/src/pages/SpecimenCollection/Source/Input.tsx --full-history --pretty=oneline --date-order --decorate=full --skip=0 --max-count=10  (completed in 0.418s)
git rev-list --full-history --count  (completed in 0.129s)
usage: git rev-list [OPTION] <commit-id>... [ -- paths... ]
  limiting output:
    --max-count=<n>
    --max-age=<epoch>
    --min-age=<epoch>
    --sparse
    --no-merges
    --min-parents=<n>
    --no-min-parents
    --max-parents=<n>
    --no-max-parents
    --remove-empty
    --all
    --branches
    --tags
    --remotes
    --stdin
    --quiet
  ordering output:
    --topo-order
    --date-order
    --reverse
  formatting output:
    --parents
    --children
    --objects | --objects-edge
    --unpacked
    --header | --pretty
    --[no-]object-names
    --abbrev=<n> | --no-abbrev
    --abbrev-commit
    --left-right
    --count
  special purpose:
    --bisect
    --bisect-vars
    --bisect-all
```

_Originally posted by @hoegge in https://github.com/DonJayamanne/gitHistoryVSCode/issues/482#issuecomment-595847367_

**Comment by ole1986** [comment]
@hoegge luckily this is another bug which we have already resolved

**Comment by hoegge** [comment]
> @hoegge luckily this is another bug which we have already resolved

Great - thanks. When / how often does the extension get updated / released so the fixes become available?

**Comment by ole1986** [comment]
Released version 0.6.2 just now!
